geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreTest.scala

```diff
@@ -72,6 +73,21 @@ class AccumuloDataStoreTest extends Specification with TestWithMultipleSfts {
       ds.getSchema(defaultSft.getTypeName) mustEqual defaultSft
     }
 
+    "create a schema with keywords" in {
```

Comment:
Given that this is in the ```AccumuloDataStoreTest```, is there any merit to making this a fuller round-trip through a MockAccumulo instance in a real data store?